### DISCUSSION TITLE
Generating a GeoPDF from QField?

### DISCUSSION THREAD

**Discussion by BoswachterMarc** [opengisch/QField#3912] [Ideas]
In QGis it is possible to export the map on your screen as a GeoPDF. This is a great tool for people working in the field without any knowledge of the mobile app.

So I wonder if this option/export could also be generated from/by qfield? This way more experienced fieldworkers could share a map/geopdf (made with QField) with less experienced fieldworkers with the sole purpose of 'seeing where they walk' and finding certain elements/features.

![Knipsel](https://user-images.githubusercontent.com/57868975/219139762-45e7ab00-3d21-40f0-b898-43fb6c562ccd.PNG)

**Comment by NickSchub** [comment]
The exported PDFs in qfield are already geoPDFs...just check it out.

**Comment by BoswachterMarc** [comment]
Really? Hadn't looked at it that way ;-) Thanks for the hint!
